docs/android/emulator/mac-with-windows-vm.md

```diff
@@ -0,0 +1,142 @@
+
+    emulator6 94105 macuser   20u  IPv4 0xa8dacfb1d4a1b51f      0t0  TCP localhost:5555 (LISTEN)
+    emulator6 94105 macuser   21u  IPv4 0xa8dacfb1d845a51f      0t0  TCP localhost:5554 (LISTEN)
+    ```
+
+    The emulator uses a pair of sequential TCP ports - an even-numbered port for console connections, and an odd-numbered port for `adb` connections. In the output above the emulator listens for `adb` on port 5555 and listens for console connections on port 5554.
+
+1. In **Terminal**, use either `nc` or `pfctl` to perform packet forwarding:
+
+    1. Use `nc` to forward inbound TCP packets received externally on port 5555 (or any other port) to the odd-numbered port on the loopback interface, and to forward the outbound packets back the other way:
+
+        ```zsh
+        cd /tmp
+        mkfifo backpipe
+        nc -kl 5555 0<backpipe | nc 127.0.0.1 5555 > backpipe
+        ```
+
+        In this example, `127.0.0.1 5555` denotes the odd-numbered port on the loopback interface.
+
+        Provided that the `nc` command keeps running in a Terminal window, packets will be forwarded as expected. Once you've finished using the emulator, you can stop `nc` performing packet forwarding by pressing <kbd>CTRL+C</kbd> in the Terminal window.
+
+        > [!NOTE]
+        > Using `nc` is often preferred to `pfctl`, particularly if the firewall is enabled on your Mac.
+
+    1. Use `pfctl` to redirect TCP packets from port 5555 (or any other port) on the [Shared Networking](https://kb.parallels.com/en/4948#section1) interface to
+    the odd-numbered port on the loopback interface:
+
+        ```zsh
+        sed '/rdr-anchor/a rdr pass on vmnet8 inet proto tcp from any to any port 5555 -> 127.0.0.1 port 5555' /etc/pf.conf | sudo pfctl -ef -
+        ```
+
+        In this example, `127.0.0.1 5555` denotes the odd-numbered port on the loopback interface.
+
+        This command sets up port forwarding using the `pf packet filter` system service. If you're using Parallels, you'll need to adjust the interface name from `vmnet8`, which is the name of the special NAT device for the Shared Networking mode in VMWare Fusion. The appropriate network interface in Parallels is most likely [vnic0](https://download.parallels.com/doc/psbm/en/Parallels_Server_Bare_Metal_Users_Guide/29258.htm).
+
+1. In your Windows VM, open **Command Prompt**.
+1. In **Command Prompt**, connect to the emulator:
+
+    ```cmd
+    adb connect ip-address-of-the-mac:5555
+    ```
+
+    Replace `ip-address-of-the-mac` in this example with the IP address of your Mac, and 5555 with the port you've used from the previous step.
+
+    > [!NOTE]
+    > Command line access to `adb` can be obtained in Visual Studio through the **Tools > Android > Android Adb Command Prompt...** menu item.  
+
+When the connection is completed Visual Studio will display the emulator as a debug target, which you can use to deploy your app to the emulator.
```

Comment:
We should note that the Mac emulator appears as a debug target in VS under `Android Local Devices` _not_ under `Android Emulators`. VS thinks it's a physical Android device - this is likely a bit surprising / non-obvious for users that newly do this, so we should direct them to look there.

For me it shows like this:
![image](https://github.com/dotnet/docs-maui/assets/245892/7bfbddd7-6f33-411a-96d4-398c468e6e27)